presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestPickTableLayout.java

```diff
@@ -47,6 +47,11 @@
     private TableHandle nationTableHandle;
     private TableLayoutHandle nationTableLayoutHandle;
 
+    public TestPickTableLayout()
```

Comment:
specify in the commit message that it's for the PickTableLayoutTest (and not generically about the tpch connector)